sources/packages/backend/apps/api/src/route-controllers/user/user.institutions.controller.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Patch,
+  UnprocessableEntityException,
+} from "@nestjs/common";
+import { BCeIDService, UserService } from "../../services";
+import BaseController from "../BaseController";
+import { UserToken } from "../../auth/decorators/userToken.decorator";
+import { IUserToken } from "../../auth/userToken.interface";
+import { BCeIDDetailsAPIOutDTO } from "./models/bceid-account.dto";
+import { InstitutionUserAPIOutDTO } from "./models/institution-user.dto";
+import { InstitutionUserPersistAPIInDTO } from "./models/institution-user-persist.dto";
+import { BCeIDAccountsAPIOutDTO } from "./models/bceid-accounts.dto";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, AllowInactiveUser } from "../../auth/decorators";
+import { ApiTags, ApiUnprocessableEntityResponse } from "@nestjs/swagger";
+import { BCeIDAccountTypeCodes } from "../../services/bceid/bceid.models";
+import { UserControllerService } from "..";
+import { ClientTypeBaseRoute } from "../../types";
+
+@AllowAuthorizedParty(AuthorizedParties.institution)
+@Controller("users")
```

Comment:
I guess that I missed but the controller name should be singular.
When not sure, take a look at swagger.